2021-10-13-github-templates.md

```diff
@@ -0,0 +1,170 @@
+# OmbuLabs Standard Github Templates
+
+## Problem:
+
+New and junior contributors might be unsure about the information we require to
+efficiently review a PR. Similarly, it might be unclear what information is required when
+a new BUG or FEATURE is logged as an issue in Github.
+
+We have existing PR and ISSUE templates but they are inconsistent and might ask for
+unnecessary information from the contributor.
+
+The goal is to simplify  and streamline the steps required to make a contribution or log
+and issue.
+
+## Proposal:
+
+Come up with a consistent set of Github templates. The templates should include prompts
+and checklists for only the ESSENTIAL pieces of information we require. We want to make
+sure that we don't ask for more information than we require - as this will discourage
+contributions.
+
+## Templates:
+
+### Pull Requests
+
+```
+    <!--- Provide a general summary of your changes in the Title above -->
+
+    ## Description
+    <!--- Describe your changes in detail -->
+    <!--- Please read the README before submitting pull requests for this project. -->
+
+    ## Motivation and Context
+    <!--- Why is this change required? What problem does it solve? -->
+    <!--- If it fixes an open issue, please link to the issue here. -->
+
+    ## How Has This Been Tested?
+    <!--- Please describe in detail how you tested your changes. -->
+    <!--- Include details of your testing environment, and the tests you ran to -->
+    <!--- see how your change affects other areas of the code, etc. -->
+
+    ## Screenshots (if appropriate):
+
+    ## Types of changes
+    <!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+    - [ ] Bug fix (non-breaking change which fixes an issue)
+    - [ ] New feature (non-breaking change which adds functionality)
+    - [ ] Breaking change (fix or feature that would cause existing functionality to change)
+
+    ## Checklist:
+    <!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+    <!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+    - [ ] My code follows the code style of this project.
+    - [ ] My change requires a change to the documentation.
+    - [ ] I have updated the documentation accordingly.
+    - [ ] I have read the **CONTRIBUTING** document.
+    - [ ] I have added tests to cover my changes.
+    - [ ] All new and existing tests passed.
+    - [ ] I will abide by the code of conduct.
+```
+
+
+### Issues: Bug Reports
+
+```
+    <!--- Provide a general summary of the issue in the Title above -->
+
+    ## Before we start...
```

Comment:
these checklists are too easy to overlook and ignore after you get used to it, I think they just add noise, I think nobody really uses them

I understand it's more like a reminder but it's not obligatory to check things, but I don't think it's adding any real value to the report